02_ShowUser.php

```diff
@@ -2,112 +2,110 @@
+			<a href="04_update_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">manage_accounts</a>
+			<a href="05_delete_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_remove</a>
 
-			</div>
-		
+		</div>
 
 
-			<!-- Title for query: show users -->
-			<div style="margin-left:8%">
-				<div class="w3-container w3-dark-grey">
-					<h2>Show User</h2>
-				</div>
+		<!-- Title for query: show users -->
+		<div style="margin-left:8%">
+			<div class="w3-container w3-dark-grey">
+				<h2>Show User</h2>
+			</div>
 
-				
-				<!-- Users php -->
-				<div class="w3-row w3-panel w3-border">
-					<div class="w3-third w3-center w3-panel">
-						<h5><user1>USERS</user1></h5>
-						  <h6>
+			<br>
+			<!-- Users php -->
+			<div class="w3-row">
+				<div class="w3-third w3-center">
+					<h5><user1>USERS</user1></h5>
+					<h6>
 						<?php
-							//connect.php (tells where to connect servername, username, password, dbaseName)
-							require "communityproject_mysqli.php";
+						//connect.php (tells where to connect servername, username, password, dbaseName)
+						require "communityproject_mysqli.php";
 
-							//create a variable to store sql code for the 'display all users' query
-							$query = ("SELECT * FROM users");
+						//create a variable to store sql code for the 'display all users' query
+						$query = ("SELECT * FROM users");
 
-							//run the query
-							$result = mysqli_query($conn,$query);
+						//run the query
+						$result = mysqli_query($conn,$query);
 
-							while ($output = mysqli_fetch_array($result))
-							{
-								echo "<user2>". $output['User_ID']."</user2><br>";
+						while ($output = mysqli_fetch_array($result))
+						{
+							echo "<user2>". $output['User_ID']."</user2><br>";
 							//closes the while loop
-								}
+						}
 						?>
-						</h6>
-					</div>
-					
-					
-					<!-- Passwords php -->
-					<div class="w3-third w3-center w3-panel">
-						<h5><password1>PASSWORDS</password1></h5>
-						<h6>
-							<?php
-							//run the query
-							$result = mysqli_query($conn,$query);
-
-							while ($output = mysqli_fetch_array($result))
-							{
-								echo "<user2>". $output['Password']."</user2><br>";
-								//closes the while loop
-							}
-							?>
-						</h6>
-					</div>
+					</h6>
+				</div>
+
+
+				<!-- Passwords php -->
+				<div class="w3-third w3-center ">
+					<h5><password1>PASSWORDS</password1></h5>
+					<h6>
+						<?php
+						//run the query
+						$result = mysqli_query($conn,$query);
+
+						while ($output = mysqli_fetch_array($result))
+						{
+							echo "<user2>". $output['Password']."</user2><br>";
+							//closes the while loop
+						}
+						?>
+					</h6>
 				</div>
 			</div>
-			
+		</div>
+
 
-		<!-- footer (sits at botto of page) -->
+		<!-- footer (sits at bottom of page) -->
 		<footer class="w3-bottom w3-center w3-black w3-padding-64">
```

Comment:
decided to keep the footer black to create that consistent theme and connection from the other pages